contrib/enum_metrics/test/test_clear_errant_enums.py

```diff
@@ -0,0 +1,27 @@
+from unittest import TestCase
```

Comment:
would it be hard to use something like this:
https://pypi.python.org/pypi/testing.cassandra
to create a dummy cassandra server and dummy data, to create some more extensive unit tests?